programs/manifest/src/state/market.rs

```diff
@@ -120,10 +120,21 @@ pub struct MarketFixed {
     /// Use at your own risk.
     quote_volume: QuoteAtoms,
 
+    // The following are informational only. All this information is redundant
+    // and could be computed with other information on the market.
+    // Total base withdrawable.
+    base_withdrawable: BaseAtoms,
+    // Total quote withdrawable.
+    quote_withdrawable: QuoteAtoms,
+    // Total base in asks. Does not include global.
+    base_in_asks: BaseAtoms,
+    // Total quote in bids. Does not include global.
+    quote_in_bids: QuoteAtoms,
+
```

Comment:
would prefer to hide this behind a feature flag, e.g. fuzzer

Reply:
Currently we have 1 market on mainnet and I plan on emptying it when rolling this out

Reply:
I dont really want it for testing only. Either push it out and have it as informational for making coingecko easier, or not at all